### DISCUSSION TITLE
[活动] OSPOCon 2022 日程出炉

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#74] [日常讨论]
OS SUMMIT NORTH AMERICA 2022 日程已经出炉，欢迎大家关注~

其中，OSPOCon 的日程如下，欢迎围观：

https://ossna2022.sched.com/type/OSPOCon/print

![image](https://user-images.githubusercontent.com/15010826/164712686-24b18cfe-f82e-4ecb-947b-5ff91b2e8858.png)
![image](https://user-images.githubusercontent.com/15010826/164712753-ed99a7b7-4142-4cc3-a5ef-571b5e6fe855.png)

**Comment by wangleo61** [comment]
谢谢王老师

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Friday, April 22, 2022 8:18 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] [活动] OSPOCon 2022 日程出炉 (Discussion #74)


OS SUMMIT NORTH AMERICA 2022 日程已经出炉，欢迎大家关注~

其中，OSPOCon 的日程如下，欢迎围观：

https://ossna2022.sched.com/type/OSPOCon/print

[image]<https://user-images.githubusercontent.com/15010826/164712686-24b18cfe-f82e-4ecb-947b-5ff91b2e8858.png>
[image]<https://user-images.githubusercontent.com/15010826/164712753-ed99a7b7-4142-4cc3-a5ef-571b5e6fe855.png>

―
